hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py

```diff
@@ -245,6 +276,12 @@ def order_book_class(self) -> CoinbaseProOrderBook:
                 await asyncio.sleep(30.0)
 
     async def listen_for_order_book_snapshots(self, ev_loop: asyncio.BaseEventLoop, output: asyncio.Queue):
+        """
+        *required
+        Subscribe to snapshot channel via web socket, and keep the connection open for incoming messages
```

Comment:
this function's name might be misleading; seems like it takes a snapshot for each trading pair and puts it in the output queue. Doesn't subscribe to a snapshot channel

Reply:
fixed.